src/GraphQL/Builders/FieldBuilder.cs

```diff
@@ -321,6 +321,25 @@ protected FieldBuilder(FieldType fieldType)
         return this;
     }
 
+    /// <summary>
+    /// Adds an argument to the field.
+    /// </summary>
+    /// <param name="type">The graph type of the argument.</param>
+    /// <param name="name">The name of the argument.</param>
+    /// <param name="configure">A delegate to further configure the argument.</param>
+    [AllowedOn<IObjectGraphType, IInterfaceGraphType>]
+    public virtual FieldBuilder<TSourceType, TReturnType> Argument(IGraphType type, string name, Action<QueryArgument>? configure = null)
```

Comment:
This overload was missing -- explicit graph type instance rather than CLR type or typeof(T) for graph type.  I've run into this issue before.  So I added it.  We have methods like this for `Field` so it seems that it was simply an oversight.

Reply:
The only question is the order of parameters.  To match the existing `Argument(Type type, string name)` I did it this way.

Reply:
It could be better to have `name` as a first argument to be consistent with a `Field` methods, but it will require changing the arguments order in `Argument(Type graphType, string name)`. Looks good to me.